plugins/cros_ec-usb/meson.build

```diff
@@ -0,0 +1,29 @@
+cargs = ['-DG_LOG_DOMAIN="FuPluginCrOSECUsb"']
+
+install_data(['cros_ec-usb.quirk'],
+  install_dir: join_paths(datadir, 'fwupd', 'quirks.d')
+)
+
+shared_module('fu_plugin_cros_ec_usb',
+  fu_hash,
+  sources : [
+    'fu-plugin-cros_ec-usb.c',
+    'fu-cros_ec-usb-common.c',
```

Comment:
Can I be a right pain? I think it should be `fu-cros-ec*' (i.e. with dashes, not underscores) to match the other plugins.

Reply:
Hi @hughsie. Thanks for the review.

Do you recommend that I rename the whole directory to be "cros-ec" instead of "cros_ec-usb"?

How about the following renames:
fu-plugin-cros_ec-usb.c -> fu-plugin-cros-ec.c
fu-cros_ec-usb-device.c/h -> fu-cros-ec-usb-device.c/h
fu-cros_ec-usb-common.c/h -> fu-cros-ec-common.h
cros_ec-usb.quirk -> cros-ec-usb.quirk


This will leave open the possibility of non-USB based updaters being written (which we have, but are not in scope right at the moment).

Reply:
I think  `fu-cros_ec-usb-device.c/h` -> `fu-cros-ec-device.c/h` is better

Reply:
And `cros_ec-usb.quirk` -> `cros-ec.quirk`

Reply:
> I think `fu-cros_ec-usb-device.c/h` -> `fu-cros-ec-device.c/h` is better

Hi @hughsie. In the future, if this plugin were to support non-USB updates would it make sense for this file to stay as"fu-cros-ec-usb-device.c"? 

Off the top of my head, there are two non-USB methods to update other cros-ec devices:
1. Google USB-C Alt Mode using a custom kernel driver
2. flashrom

Reply:
Yup, please leave this as usb if there are other methods planned.

Reply:
Done.